app/services/katello/event_daemon.rb

```diff
@@ -0,0 +1,60 @@
+module Katello
+  class EventDaemon
+    def self.runnable?
+      if defined?(Rails::Console)
+        Rails.logger.info("Not starting Katello Event Daemon in the Rails Console")
+        return false
+      end
+
+      if $PROGRAM_NAME.match(/spring/)
+        Rails.logger.info("Spring is running in the environment, not starting Katello Event Daemon")
```

Comment:
You once made me say "erratum" so I feel like I have the right to point out these comma splices.  :wink:
```suggestion
        Rails.logger.info("Spring is running in the environment; not starting Katello Event Daemon")
```

Reply:
I can appreciate that :) will update